zproject/urls.py

```diff
@@ -380,6 +381,7 @@
         "users/me/apns_device_token", POST=add_apns_device_token, DELETE=remove_apns_device_token
     ),
     rest_path("users/me/android_gcm_reg_id", POST=add_android_reg_id, DELETE=remove_android_reg_id),
+    rest_path("users/me/test_notify", POST=send_test_push_notification_api),
```

Comment:
Do we want `zulip.yaml` docs for this, or to skip them for now like these other push endpoints?

Reply:
Pushed some documentation